i18n/de.json

```diff
@@ -4457,11 +4457,7 @@
   {
     "id": "model.config.is_valid.site_url_email_batching.app_error",
     "translation": "E-Mail-Stapelverarbeitung lässt sich nicht aktivieren, wenn SiteURL nicht gesetzt ist."
-  },
```

Comment:
You should only change the `en.json` file, the rest of the translations files are handled by the translation process. Please, revert all of them except the `en.json`.